lib/src/embedded_cli.c

```diff
@@ -1092,13 +1135,22 @@ static const char *historyGet(CliHistory *history, uint16_t item) {
     return embeddedCliGetToken(history->buf, item);
 }
 
+static char *historyGetVariable(CliHistory *history, uint16_t item) {
```

Comment:
Same as another static function. Please declare it first and add definition along with other static functions